sdk/spring/spring-reference.yml

```diff
@@ -1,528 +1,754 @@
-# This file is aimed to build package reference of Azure Spring libraries to support an Intellij plugin for Azure SDK reference.
```

Comment:
How about we reserving the comments here, in case people are not clear of the meaning or rules for each property.

Reply:
this will be very difficult for an auto generated file